db/document_test.go

```diff
@@ -1,12 +1,185 @@
 package db
 
 import (
+	"bytes"
 	"encoding/binary"
+	"encoding/json"
+	"log"
 	"testing"
 
 	"github.com/couchbaselabs/go.assert"
 )
 
+var doc_1k = `{
```

Comment:
It would be possible to check this in as a file and include it into the compiled test binary using something like https://github.com/jteeuwen/go-bindata

I don't think it needs to be changed as part of this PR, but this reminded me of it.  I think it would improve test readability, and give us the flexibility to include much larger json content that would be impractical to inline in the code.